src/main/java/org/prebid/server/bidder/openx/OpenxBidder.java

```diff
@@ -119,9 +122,13 @@ private List<BidRequest> makeRequests(BidRequest bidRequest, List<Imp> bannerImp
     }
 
     private static OpenxImpType resolveImpType(Imp imp) {
-        return imp.getBanner() != null
-                ? OpenxImpType.banner
-                : imp.getVideo() != null ? OpenxImpType.video : OpenxImpType.other;
+        if (imp.getBanner() != null) {
+            return OpenxImpType.banner;
+        } else if (imp.getVideo() != null) {
+            return OpenxImpType.video;
+        } else {
+            return OpenxImpType.other;
+        }
```

Comment:
This unrelated change snuck in to resolve the last sonar warning about nested ternary